x-pack/plugins/aiops/common/api/log_categorization/create_category_request.ts

```diff
@@ -31,15 +31,16 @@ export function createCategoryRequest(
   queryIn: QueryDslQueryContainer,
   wrap: ReturnType<typeof createRandomSamplerWrapper>['wrap'],
   intervalMs?: number,
-  additionalFilter?: CategorizationAdditionalFilter
+  additionalFilter?: CategorizationAdditionalFilter,
+  useStandardTokenizer?: boolean
```

Comment:
I this should probably default to `true`

Reply:
Updated in [a666bc3](https://github.com/elastic/kibana/pull/176587/commits/a666bc3157128c12fdc3c98953781e86ee64860b).